sourcetools/buildj9tools.mk

```diff
@@ -45,8 +48,8 @@ else
 ifdef DEV_TOOLS
   JAVA_BIN  ?= $(DEV_TOOLS)/ibm-jdk-1.8.0/bin
 else
-  # requires Java 8 or 9
-  JAVA      := java8
+  # requires previous or identical Java version
+  JAVA      := java$(PREV_JAVA)
```

Comment:
If we're considering changes here, I think it's time to stop assuming users have tools called `java8` (or `java7`) available.
In fact, if `BOOT_JDK` is defined, we should simply use it instead of looking elsewhere.